preview/app/http/PreviewFilters.scala

```diff
@@ -3,18 +3,22 @@ package http
 import akka.stream.Materializer
 import common.ExecutionContexts
 import GoogleAuthFilters.AuthFilterWithExemptions
+import controllers.HealthCheck
+import googleAuth.FilterExemptions
 import model.ApplicationContext
 import play.api.http.HttpFilters
 import play.api.libs.crypto.CryptoConfig
 import play.api.mvc.{Filter, RequestHeader, Result}
 
 import scala.concurrent.Future
 
-class PreviewFilters(cryptoConfig: CryptoConfig)(implicit mat: Materializer, context: ApplicationContext) extends HttpFilters {
+class PreviewFilters(cryptoConfig: CryptoConfig, healthCheck: HealthCheck)(implicit mat: Materializer, context: ApplicationContext) extends HttpFilters {
 
+  private val exemptionsUrls = healthCheck.healthChecks.map(_.path) ++ Seq("/2015-06-24-manifest.json")
```

Comment:
Oh I assume this is why: https://github.com/guardian/frontend/blob/d13f8140c72bc30515c7da60d3ce9f150773dac8/common/app/views/fragments/metaData.scala.html#L43